app/js/helpers/jsonHelper.js

```diff
@@ -105,61 +104,4 @@ export class JSONHelper {
     });
     return colValues;
   }
-  
-  composeDescription(searchParameters) {
-    const newParameters = Object.assign({}, searchParameters);
-    let definitionKeys = Object.keys(newParameters);
-    if(definitionKeys.length === 0) {
-      return 'All Patients';
-    } else {
-      let label = '';
-      if (definitionKeys.indexOf('gender') >= 0) {
-        label = `${this.getGenderName(newParameters['gender'])} patients`;
-        delete newParameters.gender;
-        definitionKeys = Object.keys(newParameters);
```

Comment:
Could you explain a bit as to why fixing the bug required deleting this whole section?